examples/101-using-core-nuget/101-using-core-nuget.csproj

```diff
@@ -8,7 +8,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.SemanticMemory.Core" Version="0.0.230827.2-preview" />
```

Comment:
in this case please leave the version here, so we can test the nuget on this project without affecting other places.

Reply:
Do you mean explicitly define the NuGet package version? Wouldn't this defeat the purpose of centrally managing as it wouldn't take advantage of any changes done to the centrally configured package?

Reply:
FWIW I often work on this project on its own, isolated from settings outside the folder, and I find it easier to have all the settings locally. Please leave this entry here, also see other comment about not having the entry in directory.packages.props. Thanks!